pymtl3/passes/VcdGenerationPass.py

```diff
@@ -130,7 +185,8 @@ def _gen_vcd_symbol():
 
     # Vcd file takes a(0) instead of a[0]
     def vcd_mangle_name( name ):
-      return name.replace('[','(').replace(']',')')
+      # signal names with colons in it silently fail gtkwave
+      return name.replace('[','(').replace(']',')').replace(':', '_$_')
```

Comment:
hmmm why would a signal have colon in its name?

Reply:
A sliced signal is a leaf signal and has a name like `s.x[0:3]`. This happened when we tried to dump VCD for a mesh network.

Reply:
Oh, I think we don't need to dump those signals in the vcd because the non-leaf s.x will be dumped.

Reply:
Well I think the point is sometimes you do want to see a slice of some signal because that's interesting part of a concatenated vector. Plus I think this leaf slice only exists if it appears in the design. So keeping that might actually be helpful..